x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.test.ts

```diff
@@ -312,15 +312,18 @@ describe('benchmarks API', () => {
       it('should build benchmark entry agent policy and package policy', async () => {
         const packagePolicy = createPackagePolicyMock();
         const agentPolicy = createMockAgentPolicy();
-        agentPolicy.agents = 3;
 
         const cspRulesStatus = {
           all: 100,
           enabled: 52,
           disabled: 48,
         };
         const enrichAgentPolicy = await createBenchmarkEntry(
-          agentPolicy,
+          {
+            id: agentPolicy.id,
+            name: agentPolicy.name,
+            agents: 3,
+          },
```

Comment:
```suggestion
          {
            ...agentPolicy,
            agents: 3,
          },
```

to preserve the integrity of the `createMockAgentPolicy()`, and less risk of renaming properties